### PR TITLE
Replaced a uniform with weapons vendor, mapmerge

### PR DESCRIPTION
**Pull request by KwinOriginal**
## About The Pull Request

As written

## Why It's Good For The Game

More ammo for the weapons at crash

## Changelog
:cl:
add: Weapons vendor
del: uniform vendor
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Please no. This makes you unable to get spare gloves/boots if your arm is removed.
Just add a spare weapons vendor somewhere inside the Cant somewhere. Probably add one inside the CIC room. Replace one of the tables in there  with one and move the items on there somewhere else.

File: tgmc.dme [hunk 1]
```diff
@@ -1,1452 +1,1452 @@
-// DM Environment file for baystation12.dme.
```
Comment (psykzz) [inline tgmc.dme:1]: You might want to revert this file

Comment (KwinOriginal) [reply tgmc.dme:1]: Oh, I'll take a look again tomorrow, I'm just on my way to bed. Thanks

**Comment by KwinOriginal** [comment]
@Pariah919: There are already two surplus uniform vendors in the cryo bay, I think they should suffice

**Comment by KwinOriginal** [comment]
We shall meet again!

**Pull request closed without merging**